.env.template

```diff
@@ -1,17 +1,18 @@
+WEBUI_PORT=3001
 WEBUI_IMAGE=webui:latest
 RPS_IMAGE=rps:latest
 RPS_USE_VAULT=true
 RPS_USE_DB_PROFILES=true
 RPS_NODE_ENV=dev
 RPS_HTTPS=true
-RPS_DB_HOST=rpsdb
-RPS_DB_NAME=postgresdb
+RPS_DB_HOST=db
+RPS_DB_NAME=rpsdb
 RPS_DB_PORT=5432
 RPS_DB_USER=postgresadmin
 RPS_DB_PASSWORD=admin123
 RPS_WEBSOCKETTLS=true
-RPS_WEB_PORT=8081
-RPS_WEBSOCKETPORT=8080
+RPSWEBPORT=8081
```

Comment:
should these have underlines for separators like the rest?

Reply:
nope, thats was causing the issue thats linked. Purposefully NOT changing the port inside the container -- thats a no-no.